eclipse-collections-code-generator/src/main/resources/api/map/mutablePrimitivePrimitiveMap.stg

```diff
@@ -33,6 +34,17 @@ public interface Mutable<name1><name2>Map extends <name1><name2>Map, Mutable<nam
 {
     void put(<type1> key, <type2> value);
 
+   /**
+     * This method allows Mutable<name1><name2>Map the ability to add an element in the form of <name1><name2>Pair.
+     *
+     * @see #put(<type1>, <type2>)
+     * @since 9.1.0
+     */
+    default void putPair(<name1><name2>Pair keyValuePair)
```

Comment:
This should return the value to be consistent with put.

Reply:
Same answer as above